lib/octicons_helper/test/octicons_helper_test.rb

```diff
@@ -7,7 +7,7 @@
     end
 
     it "renders the svg" do
-      assert_match /<svg.*octicon-x.*><path.*\/><\/svg>/, octicon_v2("x")
+      assert_match /<svg.*octicon-x.*>.*<\/svg>/, octicon_v2("x")
```

Comment:
This test was failing because the new generated SVGs use a non-self-closing `<path>` element (`<path></path>` instead of `<path />`). We probably shouldn't require that SVGs only have a single self-closing `<path>`. I updated the test to allow anything between the opening and close `<svg>` tags.